pipelines/apDetectorVisitQualityCore.yaml

```diff
@@ -79,6 +79,19 @@ tasks:
         templateCoveragePercent: percent
       python: |
         from lsst.analysis.tools.atools import DiffimMetadataMetricTool
+  associationCore:
+    class: lsst.analysis.tools.tasks.MetadataAnalysisTask
+    config:
+      connections.taskName: diaPipe
+      connections.outputName: diaPipe_metadata
+      metadataDimensions: ["instrument", "visit", "detector"]
+      atools.calexpMetadataMetrics: MetadataMetricTool
+      atools.calexpMetadataMetrics.taskName: diaPipe
+      atools.calexpMetadataMetrics.metrics:
```

Comment:
`atools.calexpMetadataMetrics` as used here is just a label, which references `MetadataMetricTool` and allows it to be configured. You can name it anything you want, so I suggest renaming to `atools.diaPipeMetadataMetrics`